lxd/db/cluster/update.go

```diff
@@ -44,7 +45,15 @@ var updates = map[int]schema.Update{
 }
 
 func updateFromV10(tx *sql.Tx) error {
+	// Before doing anything save the counts of all tables, so we can later
+	// check that we don't accidentally delete or add anything.
+	counts1, err := query.CountAll(tx)
+	if err != nil {
+		return errors.Wrap(err, "failed to count rows in current tables")
```

Comment:
Capitalization

Reply:
Fixed.

Reply:
Was that comment supposed to indicate that `fixed` should be `Fixed`? If so it doesn't look `Fixed` to me. :)

Reply:
Opps, true! Should be ```really-fixed``` now.